app/views/main.scala.html

```diff
@@ -59,17 +59,23 @@
         @content
 
     </div>
-
-    @if(url.get == "/audit") {
-
-        <script type="text/javascript">
-            document.getElementsByTagName("body")[0].style.overflow = "hidden";
-            document.getElementById("wrap").style.padding = "0 0 35px";
-        </script>
+    @if(url.get != "/mobile") {
+        <head>
```

Comment:
What does this new piece do?

Reply:
it fixes an issue having to do with the page loading in zoomed in

Reply:
Okay, it didn't fix it on my phone so I added a new fix. Could you test out removing this fix and seeing if it still works on your phones?

Reply:
Removed and tested works fine!